TCA Practice/Root/UserView/EditUserView.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  EditUserView.swift
+//  TCA Practice
+//
+//  Created by xiangmu shi on 2022/7/19.
+//
+
+import ComposableArchitecture
+import SwiftUI
+
+struct EditUserView: View {
+    let store: Store<UserState, UserAction>
+
+    private enum Constants {
+        static let leftTextWidth: CGFloat = 85
+        static let rightTextWidth: CGFloat = 120
+        static let rightTextMaxWidth: CGFloat = 180
+    }
```

Comment:
In general we try to let SwiftUI layout system to be as flexible as we can and try to avoid fixed frames. There are some tools at our disposal such as `layoutPriority` and `alignmentGuide` to have more control over the resulting layout. One last resort is to use `GeometryReader`. This view allows you to pass a closure that takes a `GeometryReaderProxy` and from there you can get the frame of the container view and define relative sizes to your children.

Reply:
cool , That is good for SwiftUI, 
fixed!